src/settings-ui/Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -168,6 +168,9 @@ public GeneralViewModel(ISettingsRepository<GeneralSettings> settingsRepository,
             string etwDirPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "Microsoft\\PowerToys\\etw");
             DeleteDiagnosticDataOlderThan28Days(etwDirPath);
 
+            string localLowEtwDirPath = Path.Combine(Environment.GetEnvironmentVariable("USERPROFILE"), "AppData", "LocalLow", "Microsoft", "PowerToys", "etw");
+            DeleteDiagnosticDataOlderThan28Days(localLowEtwDirPath);
```

Comment:
By the way, should we add somewhere in the UI that this data gets deleted after 28 days? That might be necessary info?